ydb/public/lib/ydb_cli/commands/tpch_schema.sql

```diff
@@ -24,14 +24,14 @@ CREATE {external} TABLE `{path}lineitem` (
     l_linenumber Int32 {notnull},
```

Comment:
I think linenumber could be also bigger than Int32

Reply:
No, linenumber is very small, at most like 10 or so